### ISSUE TITLE
The modify amount key of patterns is force bounded as `Middle-Click` key

### ISSUE THREAD

**Issue by zyxkad** [AppliedEnergistics/Applied-Energistics-2#7281]
### Describe the bug

The modify item amount key is force bounded to the `Middle button`, but it should as same as the `Pick Block` key

<img width="959" alt="Screenshot 2023-07-20 at 3 11 36 PM" src="https://github.com/AppliedEnergistics/Applied-Energistics-2/assets/91082411/c83ac85d-ae0d-4f4c-9866-dc7fcb5be24b">

![image](https://github.com/AppliedEnergistics/Applied-Energistics-2/assets/91082411/2449e86f-50d1-40ea-8bd8-b019ed2cd765)

### How to reproduce the bug

1. Set the `Pick Block` key to something other than `Middle button`
2. Open a pattern terminal, put something in the pattern
3. there will be hint letting you to use `Middle button` than the actual `Pick Block` key

### Expected behavior

The set amount key should be as same as the `pick block` key

### Which minecraft version are you using?

1.19

### On which mod loaders does it happen?

Forge

### Crash log

N/A

**Comment by GhettoKoala** [comment]
Yeah i wish we could rebind this, my middle mouse click hasnt worked for years now and I'm coming into a few issues with AE2 forcing you to use MM, i saw in another post it seems to be hard coded to be bound to the mouse? Like if you set pick block to left click it will bring up the modify amount menu which does work as a temporary work around.
